source/_patterns/04-pages/03-multioffer-teaser.json

```diff
@@ -73,36 +73,36 @@
       "multiOffer": [
         {
           "h5Content": "Calabrië ",
-          "region": {"content": ""},
-          "hotel": {"content": "LABRANDA Ephesus Premium "},
-          "info": {"content": "22 Wochen | AI+ | nur Hotel"},
+          "region": {"pContent": ""},
+          "hotel": {"pContent": "LABRANDA Ephesus Premium "},
+          "info": {"pContent": "22 Wochen | AI+ | nur Hotel"},
           "linkedPriceCurrency": "€",
           "linkedPriceInfo": "p.p vanaf",
           "linkedPriceValue": "4.859"
         },
         {
           "h5Content": "Ramada Plaza Jumeirah Beach ",
-          "region": {"content": ""},
-          "hotel": {"content": "LABRANDA Blue Bay "},
-          "info": {"content": "222 Wochen | AI+ | nur Hotel"},
+          "region": {"pContent": ""},
+          "hotel": {"pContent": "LABRANDA Blue Bay "},
+          "info": {"pContent": "222 Wochen | AI+ | nur Hotel"},
```

Comment:
222? is it right?

Reply:
Placeholder text. It's not right. 2 weeks would be more meaningful. Thanks, you just reminded me to write a Todo, for a handover of the feature branch. I'll make this one more meaningful.